lms/templates/dashboard/_dashboard_certificate_information.html

```diff
@@ -52,7 +52,7 @@
   </p>
 % endif
 
-% if cert_status['show_disabled_download_button'] or cert_status['show_download_url'] or cert_status['show_survey_button']:
+% if enrollment.mode != 'audit' and (cert_status['show_disabled_download_button'] or cert_status['show_download_url'] or cert_status['show_survey_button']):
```

Comment:
so - in essence - this just hides certificates that will have been generated for "audit" learners, until we - in the future - don't generate such certificates. Right?